js/ragas.ts

```diff
@@ -603,6 +603,11 @@ export const AnswerRelevancy: ScorerWithPartial<
   string,
   RagasArgs & {
     strictness?: number;
+    /**
+      @default
+      If not provided, the default model of {@link EmbeddingSimilarity} is used.
+     */
+    embeddingModel?: string;
```

Comment:
would you mind making the same change in python?

https://github.com/braintrustdata/autoevals/blob/main/py/autoevals/ragas.py#L833

Reply:
> would you mind making the same change in python?
> 
> https://github.com/braintrustdata/autoevals/blob/main/py/autoevals/ragas.py#L833

can do, though on further inspection it looks like there are a few other Ragas evaluators that require embedding models as well as LLMs. will apply this fix to them as well.